pkg/standalone-utils/test/ProtocolIdRegistry.test.ts

```diff
@@ -99,4 +102,36 @@ describe('ProtocolIdRegistry', () => {
       expect(await registry.isValidProtocolId(MAX_UINT256)).to.equal(false);
     });
   });
+
+  describe('renaming protocol IDs', async () => {
+    const newProtocolName = 'Test Protocol';
+    let transactionReceipt: ContractReceipt;
+
+    context('authorized user', async () => {
+      sharedBeforeEach('rename protocol', async () => {
+        transactionReceipt = await (await registry.connect(authorizedUser).renameProtocolId(0, newProtocolName)).wait();
```

Comment:
Can we add a check that reads protocol ID 0 and verifies that the existing name is different to the new one, before actually renaming it? (This would be part of the `sharedBeforeEach`, since it's just a check for readability and not part of the test).